themes/10up-theme/assets/css/frontend/base/wordpress.css

```diff
@@ -50,3 +50,35 @@
 
 	@apply max-w-none;
 }
+
+/**
+	* Post title styles
+	*
+	* These will be applied to all headings with an `entry-title` class on the
+	* frontend and to the post heading in the block editor.
+	*
+	* The supplied styles are meant to match the default `h1` classes from
+	* Tailwind Typography.
+*/
+.edit-post-visual-editor__post-title-wrapper .editor-post-title__input,
+.entry-title {
+
+	@apply max-w-content mx-auto mb-6 text-6xl font-extrabold;
```

Comment:
@Nikki-Jones @iansvo - do we want to be opinionated with these rules? Or should we just leave them untouched in terms of what `@tailwind/typography` provides?

Reply:
@iansvo @dainemawer I don't see the harm in being opinionated about this considering it will likely be overwritten and can be a starting point for anyone doing so

Reply:
@dainemawer I think Nikki is on the right track with providing a style for this in this manner. Overall I think we'd want to make sure that the value is correct for the normal typography sizing we use (e.g. if we're using prose-xl most of the time we'd probably want to use whatever that H1 font-size is) but we _do not_ want to just include `prose` classes all over. 

Those classes are not safe to drop on a big container that may include things that shouldn't receive those styles. The styles are written in such a way that it assumes you're placing them on the parent container of some long form content (think like blog post body, WYSIWYG editor output, that kinda stuff.

Reply:
Makes sense!